src/parser/mage/fire/modules/features/CombustionFirestarter.js

```diff
@@ -44,8 +45,8 @@ class CombustionFirestarter extends Analyzer {
     }
 
     this.combustionCast = false;
-    const healthPercent = event.hitPoints / event.maxHitPoints;
-    if (healthPercent > FIRESTARTER_HEALTH_THRESHOLD) {
+    if (event.hitPoints > 0) {this.healthPercent = event.hitPoints / event.maxHitPoints;}
```

Comment:
please don't do single line if-statements, they make reading code harder

Reply:
ill make a new pr to fix it.